norfair/tracker.py

```diff
@@ -361,10 +406,37 @@ def conditionally_add_to_past_detections(self, detection):
             detection.age = self.age
             self.past_detections.append(detection)
 
+    def update_embeddings_ema(self, detection):
+        """Calculates an EMA over the TrackedObject's past detections embeddings"""
+        if detection.embedding is not None:
+            if self.embeddings_ema is not None:
+                self.embeddings_ema = (
+                    self.alpha * self.embeddings_ema
+                    + (1 - self.alpha) * detection.embedding
+                )
+            else:
+                self.embeddings_ema = detection.embedding
```

Comment:
This should probably be defined inside a demo instead of as a core component of Norfair. Also maybe a similar function to this one is defined in numpy/scipy, and we could use it instead.